<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_2</class>
 <widget class="QMainWindow" name="MainWindow_2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>768</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Mail System</string>
  </property>
  <property name="iconSize">
   <size>
    <width>34</width>
    <height>17</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <widget class="QListWidget" name="revMail_list">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>241</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="revMail_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>34</width>
      <height>17</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/revMail.png</normaloff>
      <normalon>../images/revMail.png</normalon>../images/revMail.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QTextBrowser" name="revMail_text">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>531</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="writeMail_button">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>34</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../images/writeMail.png</normaloff>
      <normalon>../images/writeMail.png</normalon>../images/writeMail.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>34</width>
      <height>17</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="account_label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>481</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="revMailnum_lable">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>10</y>
      <width>141</width>
      <height>18</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
